hdfs/client.py

```diff
@@ -62,7 +73,7 @@ def __init__(self, method, **kwargs):
     self.kwargs = kwargs
 
   def __call__(self):
-    pass # make pylint happy
+    pass  # make pylint happy
```

Comment:
I'm guessing you're using a linter which clashes with the convention in the code: can you revert the double-space before comment changes (here and in a couple other places below)?